CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Table of Contents
 ### New Features
 
 * The `ProjectLayout` may now retrieve the test source root for the current project in order to allow for test healing to be applied. This is required for custom `ProjectLayouts` to be implemented, in order for this feature to work. For more information, please refer to the documentation.
+* Use `import: $reference` to import another filter into an filter or an ignore file. Ignoring all positioning changes is now easily possible with `import: positioning.filter`. Whole cascades and hierarchies of filters can now be created this way. 
```

Comment:
Do we need to make this more future proof?

``` 
import: name=positioning(.filter)?
import: url=https://
import: url=file://
import: file=/Users/with/absolute/path
import: file=relative/path/to/project
```

Otherwise please stick to the given syntax:

- `:` should delegate to another loader. You could argue that this is the case but it could possible ruin/make it more difficult to implement the rules above (if we deem necessary). 
- `=` should be responsible for loading. We could make this then default/shortcut for `import: filter=${name}`

Reply:
I think the syntax should be defined from a users perspective. Whether an implementation delegates to another loader is an implementation detail that the user doesn't care and shouldn't need to care about and that can be changed if needed, i.e. it's not a strong necessity but merely "accidental" due to a well-thought design.

As for the proposed extension (name/url/file), I don't see the benefit. As you already show by the examples, `url` and `file` are somewhat redundant, as `url` is a superset of `file`. And I don't see what benefit `name` yields. So I oppose that suggestion.